core/org.wso2.carbon.core.services/src/main/java/org/wso2/carbon/core/services/util/CarbonAuthenticationUtil.java

```diff
@@ -76,7 +76,7 @@ public static void onFailedAdminLogin(HttpSession httpSess, String username, int
            msg +=  " from IP address " + remoteAddress;
         }
         log.warn(msg);
-        if (!isLegacyAuditLogsDisabled()) {
+        if (!isEnableV2AuditLogs()) {
             audit.warn(msg);
```

Comment:
Are the V1 audit logs also printed as warn logs? They should be info logs right?

Reply:
Added a git issue to track https://github.com/wso2/product-is/issues/24356